x/ibc/07-tendermint/types/header.go

```diff
@@ -37,10 +37,11 @@ func (h Header) GetHeight() exported.Height {
 	if h.Header == nil {
 		return clienttypes.ZeroHeight()
 	}
-
-	// Enforce clienttypes.Height to use 0 epoch number
-	// TODO: Retrieve epoch number from chain-id
-	return clienttypes.NewHeight(0, uint64(h.Header.Height))
+	epoch, err := clienttypes.ParseChainID(h.Header.ChainID)
+	if err != nil {
+		return clienttypes.Height{}
```

Comment:
```suggestion
		return clienttypes. ZeroHeight()
```

Reply:
Think the convention is to use `Height{}` when returning as a result of an error correct? cc: @colin-axner

Reply:
I think either way is fine. I think that convention is preferred when you are explicitly returning an error. Since we cannot return an error here, it might be useful to just say the zero height implies an error occured